deploy-android-app/action.yml

```diff
@@ -109,16 +122,16 @@ runs:
     - name: Increase version code and change version name
       shell: bash
       run: |
-        echo "${{ env.RELEASE_VERSION }}" > VERSION
-        current_version_code=$(cat VERSION_CODE)
+        echo "${{ env.RELEASE_VERSION }}" > ${{ inputs.version_name_filepath }}
+        current_version_code=$(cat ${{ inputs.version_code_filepath }})
```

Comment:
is cat needed?

Reply:
Yes its needed because that is a file path its reading from not just the value.